src/components/templates/PartyMap/components/Map/MapRoom.tsx

```diff
@@ -56,48 +65,68 @@ export const MapRoom: React.FC<MapRoomProps> = ({ venue, room }) => {
 
   const selectRoomWithSound = useCallback(
     (e: React.MouseEvent<HTMLElement, MouseEvent>) => {
+      console.log("TFARRR");
       if (!shouldBeClickable) return;
       analytics.trackEnterRoomEvent(room.title, room.template);
       isExternalPortal(room) ? openUrl(room.url) : enterWithSound();
     },
     [analytics, enterWithSound, room, shouldBeClickable]
   );
 
+  const [referenceElement, setReferenceElement] = useState<HTMLElement | null>(
+    null
+  );
+  const [popperElement, setPopperElement] = useState<HTMLElement | null>(null);
+  const { styles: popperStyles, attributes: popperAttributes } = usePopper(
+    referenceElement,
+    popperElement
+  );
+
+  const popoverContainerElement = document.querySelector("#popoverContainer");
+
   return (
-    <div
-      className={styles.MapRoom}
-      style={roomInlineStyles}
-      /*onClick={shouldBeClickable ? selectRoomWithSound : undefined}
-    onMouseEnter={shouldBeClickable ? handleRoomHovered : undefined}
-    onMouseLeave={shouldBeClickable ? handleRoomUnhovered : undefined}*/
-    >
+    <div className={styles.MapRoom} style={roomInlineStyles}>
       <div className={styles.PortalOnMap}>
         <div className={styles.PortalImage} onClick={selectRoomWithSound}>
           <img src={room.image_url} alt={room.title} />
         </div>
-        <div className={styles.PortalTitle}>
-          {room.title}{" "}
-          <span>
-            <span></span>
-            <RoomAttendance room={room} />
-          </span>
-          {/* TODO Make the info icon display info */}
-          <button className={styles.InfoButton}>
-            <span />
-          </button>
+        <div className={styles.portalInfo}>
+          <div className={styles.PortalTitle}>
+            {room.title}
+            <span>
+              <span></span>
+              <RoomAttendance room={room} />
+            </span>
+            <span
+              className={styles.InfoButton}
+              ref={setReferenceElement}
+              onClick={toggleInfoVisible}
+            >
+              <span />
+            </span>
+          </div>
+          {infoVisible &&
+            popoverContainerElement &&
+            ReactDOM.createPortal(
+              <div
+                className={styles.PortalPopupInfo}
+                ref={setPopperElement}
+                style={popperStyles.popper}
+                {...popperAttributes.popper}
+              >
+                <h3>TODO Title</h3>
+                <p>TODO</p>
+                <span
+                  className={styles.PortalInfoButton}
+                  onClick={selectRoomWithSound}
+                >
+                  Enter
+                </span>
+              </div>,
+              popoverContainerElement
+            )}
```

Comment:
Maybe this `ReacDOM.createPortal()` call can be extracted in own popover component, have some nice comment about the need for it etc. In this place it would look like
```tsx
<Popover>
  <div> ... </div>
</Popover>
```

Reply:
Potentially. I've created a placeholder issue to track this: https://github.com/sparkletown/internal-sparkle-issues/issues/1748